rest/model/task_annotations.go

```diff
@@ -155,3 +160,24 @@ func ArrAPIIssueLinkArrtaskannotationsIssueLink(t []APIIssueLink) []annotations.
 	}
 	return m
 }
+
+func GetJiraTickets(issuelinks []APIIssueLink) ([]*APIIssueLink, error) {
+	settings := evergreen.GetEnvironment().Settings()
+	jiraHandler := thirdparty.NewJiraHandler(*settings.Jira.Export())
+	catcher := grip.NewBasicCatcher()
+
+	var res []*APIIssueLink
+	for _, issue := range issuelinks {
+		urlObject, err := url.Parse(*issue.URL)
+		catcher.Wrap(err, "problem parsing the issue url")
+		if urlObject != nil && urlObject.Host == "jira.mongodb.org" {
+			jiraIssue, err := jiraHandler.GetJIRATicket(*issue.IssueKey)
+			catcher.Wrap(err, "error getting Jira ticket")
+			if jiraIssue != nil {
+				issue.JiraTicket = jiraIssue
```

Comment:
Really you could just assign this regardless of whether or not jiraIssue is nil, issue.JiraTicket would've been nil already (i.e. you could just assign directly to issue.JiraTicket at 174)